src/runtime_src/core/edge/drm/zocl/zocl_ert.c

```diff
@@ -61,7 +61,7 @@ static int zocl_ert_probe(struct platform_device *pdev)
 	}
 	ert->hw_ioremap = map;
 	ert_info(pdev, "IP(embedded_scheduler_hw) IO start %llx, end %llx",
-			      res->start, res->end);
+	      (unsigned long long)res->start, (unsigned long long)res->end);
```

Comment:
size_t is long unsigned int type, would it make more sense to use "lx" and cast it to (unsigned long) ?

Reply:
agree